application/server.go

```diff
@@ -26,6 +30,17 @@ func New() (*fiber.App, error) {
 	panic(wire.Build(wire.NewSet(NewServer, logger.Set, configs.Set, db.Set)))
 }
 
+func NewMusicsController() (controller.MusicsController, error) {
```

Comment:
https://github.com/Nexters/myply/pull/40/files#r928938915
This is a continuation of the comment above.

I think this should be moved to musics_controller.go
Also music router needs provider(wire.Set(NewRouter...))